tests/http_spec.rs

```diff
@@ -157,12 +161,17 @@ impl HttpSpec {
         .init();
     });
 
-    let spec = match source {
+    let spec: HttpSpec = match source {
       Source::Json => anyhow::Ok(serde_json::from_str(&contents)?),
       Source::Yml => anyhow::Ok(serde_yaml::from_str(&contents)?),
       _ => Err(anyhow!("only json and yaml are supported")),
-    };
-    anyhow::Ok(spec?)
+    }?;
+
+    spec.env.iter().for_each(|(key, value)| {
+      std::env::set_var(key, value);
+    });
```

Comment:
We shouldn't need this anymore. Set it `http_spec::run` inside of the server_context.

Reply:
done